py/objgenerator.c

```diff
@@ -238,4 +251,7 @@ const mp_obj_type_t mp_type_gen_instance = {
     .getiter = mp_identity_getiter,
     .iternext = gen_instance_iternext,
     .locals_dict = (mp_obj_dict_t*)&gen_instance_locals_dict,
```

Comment:
A minor, but most types have .locals_dict as the last line, would be nice to follow that.